lib/feature_flags.js

```diff
@@ -31,7 +30,8 @@ exports.released = [
   'serverless_mode',
   'send_request_uri_attribute',
   'synthetics',
-  'dt_format_w3c'
+  'dt_format_w3c',
+  'fastify_instrumentation'
```

Comment:
I think we are fine given this is instrumentation / there aren't many experimental bits but calling out historically we'll default the feature flag on before removing (usually with a major release) as it allows people to turn things back off if unexpected issues arise. 

For instrumentation though, that doesn't seem like an issue as we typically add as a new feature w/o a feature flag to start. This case just started experimental and doesn't seem like the sort of thing you'd need/want to turn off (outside of the generic potential future want to enable/disable instrumentation once we resolve state tracking).